### ISSUE TITLE
The number of epochs affects the performance of the model

### ISSUE THREAD

**Issue by jh072535** [MrChenFeng/MaskCon_CVPR2023#1]
Thanks for your sharing of MaskCon. 
However, as I try to applied Cars196 on your method, I found it There was a decrease in model accuracy (Epoch=200). However, I found that epoch=30/50 is correct, which the performance of epoch=50 is equal to your paper indicator. Could you tell me the reason of this impunity, or do you have this impunity in your training time?
Thanks!

**Comment by MrChenFeng** [comment]
Hi, I didn't notice this before. Please ensure you strictly follow the same hyperparamters:
![1693867680761](https://github.com/MrChenFeng/MaskCon_CVPR2023/assets/24207392/851c1c3a-cf61-4f71-a962-917884daf4f6)
